--- install-tl-ubuntu	2017-03-14 19:26:04.537106526 -0300
+++ install-tl-ubuntu.dockerized	2017-03-27 14:44:12.283840123 -0300
@@ -296,7 +296,7 @@
   echo "${ECHOPREFIX}checking for TeX Live environment variables..."
   # This list is from the output of 'texconfig conf'
   grepList="TEXMFMAIN\|TEXMFDIST\|TEXMFLOCAL\|TEXMFSYSVAR\|TEXMFSYSCONFIG\|TEXMFVAR\|TEXMFCONFIG\|VARTEXFONTS\|TEXMF\|SYSTEXMF\|TEXMFDBS\|WEB2C\|TEXPSHEADERS\|TEXCONFIG\|ENCFONTS\|TEXFONTMAPS"
-  BAD_VARS=$( sudo -u "${username}" bash --login -c env | grep "${grepList}" )
+  BAD_VARS=$( bash --login -c env | grep "${grepList}" )
   # I tried to get the environment variables set using bash -i but this creates problems
   # so I will at least grep .bashrc (although if .bashrc sources another file this will
   # not pick that up)
